src/test/java/Homework17jimmyvu.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17jimmyvu extends BaseTest{
+    @Test(enabled = true, priority = 1, description = "add a song to a playlist")
+    public static void addSongToPlaylist() throws InterruptedException {
+        navigateToPage();
+        Thread.sleep(1000);
+        login("jimmypvu@gmail.com", "te$t$tudent");
+        Thread.sleep(1000);
+        getASong();
+        Thread.sleep(1000);
+
+        WebElement song = driver.findElement(By.xpath("//section[@id='playlistWrapper']/div[@class='song-list-wrap main-scroll-wrap playlist']/div[@class='virtual-scroller scroller']/div[@class='item-container']/table[@class='items']/tr[@class='song-item']/td[contains(text(), 'Waiting On A Train')]"));
+        Assert.assertTrue(song.isDisplayed());
```

Comment:
Please, use `assertEquals` will give you more information for example if you compare expectedNameSong and actualNameSong, it will show exact match. But assertTrue will only show the result, like expected true, found false

For exampe  `Assert.assertEquals(song.getAttribute("innerHTML"),"HoliznaCC0 - Waiting On A Train");`